test/services/helpers/services/submissions.js

```diff
@@ -4,12 +4,12 @@ let createdSubmissions = [];
 
 const create = (opt) => async (data) => {
 	data.schoolId = data.schoolId || opt.schoolId;
-	if (!data.homeworkId || !data.studentId) {
-		throw new Error('testSubmission requires a homework and student id!');
-	}
-	const homework = await submissionModel.create(data);
-	createdSubmissions.push(homework._id);
-	return homework;
+	data.homeworkId = data.homeworkId || opt.generateObjectId();
```

Comment:
is the generated hpmeworkId matching an existing homework? the model requires it is referenced and required and should not be just any id. same for studentId. doesn't it fail?